ileap-data-model/schemas/tad.json

```diff
@@ -33,66 +50,70 @@
         }
       ]
     },
-    "loadFactor": {
+    "energyCarrier": {
+      "description": "see https://sine-fdn.github.io/ileap-extension/#element-attrdef-tad-energycarrier",
       "anyOf": [
         {
-          "$ref": "#/definitions/Decimal"
+          "$ref": "#/definitions/EnergyCarrierType"
         },
         {
           "type": "null"
         }
       ]
     },
-    "emptyDistanceFactor": {
+    "energyCarriers": {
+      "description": "see https://sine-fdn.github.io/ileap-extension/#element-attrdef-tad-energycarriers",
       "anyOf": [
         {
-          "$ref": "#/definitions/Decimal"
+          "$ref": "#/definitions/NonEmptyVec_for_EnergyCarrier"
```

Comment:
The type name here should be `EnergyCarriers`, no? I think this is the issue we have seen before with the json schema generation. (Also occurring on line `379)`

Reply:
Good catch! 
Let me look at this before merging this PR in.